src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
The supportedFramework entry should be updated with what conrete frameworks you expect that projectreference to support.  Since the project is supporting netstandard2.0 that means... the frameworks that support netstandard2.0: netcoreapp2.0;net461;$(AllXamarinFrameworks).

It's important that you understand this part if you intend to use it.  The purpose of SupportedFramework is for the developer of the package to record in source control where he/she expects that asset to apply.  It's a *safegaurd* so that when folks retarget something and it changes the applicability of API versions in their package they do so with the explicit knowledge of what that change will do.  Please read more here: https://github.com/dotnet/wcf/pull/1871#issuecomment-297492616